### ISSUE TITLE
Migrate snapshot test for internet-header

### ISSUE THREAD

**Issue by oliverschuerch** [swisspost/design-system#1679]
```[tasklist]
### Tasks
- [ ] blocked by #1700
- [ ] copy & paste packages/documentation/cypress/snapshots/internet-header/header.snapshot.ts to packages/documentation-v7/cypress/snapshots/internet-header/header.snapshot.ts
- [ ] copy & paste packages/documentation/cypress/snapshots/internet-header/footer.snapshot.ts to packages/documentation-v7/cypress/snapshots/internet-header/footer.snapshot.ts
- [ ] copy & paste packages/documentation/cypress/snapshots/internet-header/breadcrumb.snapshot.ts to packages/documentation-v7/cypress/snapshots/internet-header/breadcrumb.snapshot.ts
- [ ] update all urls to visit
```

**Comment by imagoiq** [comment]
It looks like already done, please reopen if it's not the case.
